components/checkbox/checkbox.css

```diff
@@ -1,4 +1,16 @@
 .mt-checkbox {
+  /*
+    This (ugly) set of variables defines the variable fallback implementation.
+
+    1. If `mt-checkbox-backgroundColor` is set, then that will be applied
+    2. Otherwise, `mt-mainColor` will be applied
+    3. Otherwise, `#2196f3` will be applied
+  */
+  --_mt-checkbox-backgroundColor: var(--mt-checkbox-backgroundColor, var(--mt-mainColor, #2196f3));
+  --_mt-checkbox-checkColor: var(--mt-checkbox-checkColor, #fff);
+  --_mt-checkbox-borderRadius: var(--mt-checkbox-borderRadius, 2px);
+  --_mt-checkbox-borderWidth: var(--mt-checkbox-borderWidth, 2px);
```

Comment:
I am not sure about the borderRadius and the borderWidth being customizable. I think we should just allow to change colors for now. What do you think?

Reply:
Agreed. I'll remove them.